knpu/behat-hooks-background.md

```diff
@@ -98,20 +130,32 @@ Man what a great looking scenario, let's run it!
 vendor/bin/behat features/ls.feature
 ```
 
-As expected it now says there's one missing step definition. Copy the PHP code
-that prints out into `FeatureContext`. Remove the throw exception line, and update
-the `arg1`'s to `dir`. Inside the function use `mkdir($dir)` to actually make that
-directory. Simple!
+As expected it now says there's one missing step definition. Copy the PHP code that prints
+out into `FeatureContext`. Remove the throw exception line, and update the `arg1`'s to `dir`:
+
+[[[ code('94c81b87f3') ]]]
+
+Inside the function use `mkdir($dir)` to actually make that directory:
+
+[[[ code('7342101bd8') ]]]
```

Comment:
I realized that in our code, we called these `$dirName` unfortunately. This looks like a more obvious inconsistency when you read the script (since the working refers to `$dir`), so I updated the code itself to match this and updated the code blocks. I try not to do that too often, because it's still kind of a pain to do those updates :)